modules/actions/pool/sync-join-exits.ts

```diff
@@ -52,60 +52,42 @@ export const syncJoinExits = async (vaultSubgraphClient: V3VaultSubgraphClient,
     const events = joinExits.filter((event) => !existingEvents.some((existing) => existing.id === event.id));
```

Comment:
do we need this? we dont need it for swaps, think we can omit that here as well? (also for v2)

Reply:
removed